src/components/20-molecules/top-content-bar/index.js

```diff
@@ -46,6 +52,7 @@ class AXATopContentBar extends LitElement {
   }
 
   firstUpdated() {
+    this.inlineStyles('childStyles');
```

Comment:
Similarly, I'd like to see a brief comment on what kinds of things `childStyles` might do.

Reply:
Isn't it sufficient to declare it at class level?

Reply:
ok, valid